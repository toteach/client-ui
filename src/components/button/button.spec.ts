import { mount, Wrapper } from '@vue/test-utils';
import Vue from 'vue';

import TButton from './button.vue';

describe('TButton', () => {
  let wrapper: Wrapper<Vue>;
  beforeEach(() => {
    wrapper = mount(TButton, {
    });
  });
  it('is initialized', () => {
    expect(wrapper.name()).toBe('TButton');
    expect(wrapper.isVueInstance()).toBeTruthy();
  });
  it('renders correctly', () => {
    expect(wrapper.html()).toMatchSnapshot();
  });
});
