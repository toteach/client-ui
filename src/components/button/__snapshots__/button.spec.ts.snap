// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`TButton renders correctly 1`] = `
"<button class=\\"t-button\\">
  <!---->
  <!----></button>"
`;
